pom.xml

```diff
@@ -532,6 +532,11 @@
           <artifactId>java-json-canonicalization</artifactId>
           <version>1.1</version>
         </dependency>
+        <dependency>
```

Comment:
I'm just putting this comment here in a random place but I think we should have a release note snippet. It's a nice feature to support more than 50,000 datasets!

Reply:
There is now a snippet. Thanks!